db/migrate/20220613110734_drop_sites_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropSitesTable < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured { drop_table :sites }
```

Comment:
Can this be done in the same PR as the code changes? ie. does it have any code dependencies which might error if the migration and code deployment don't happen in the expected order?